charts/alfresco-connector-ms365/values.yaml

```diff
@@ -0,0 +1,44 @@
+global:
+  alfrescoRegistryPullSecrets: quay-registry-secret
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+replicaCount: 2
+nodeSelector: {}
+image:
+  repository: quay.io/alfresco/alfresco-ooi-service
+  tag: 1.1.3
+  pullPolicy: IfNotPresent
+  internalPort: 9095
+service:
+  name: ms365-service
+  type: ClusterIP
+  externalPort: 80
+ingress:
+  path: /ms365-service
```

Comment:
I think this can break the pipeline as it may require config changes at the Office365 app config level